lib/src/kbasesearchengine/events/StoredStatusEvent.java

```diff
@@ -18,36 +21,21 @@
     private final StatusEventProcessingState state;
     private final Optional<Instant> updateTime;
     private final Optional<String> updater;
+    private final Set<String> workerCodes;
```

Comment:
It will be helpful if you can document what workerCodes are and what they are being used for. 

Same for updater. The other attribs are quite unambiguous.

Reply:
Is the javadoc for the getters and builder with() methods not clear? I just reread them and they seem ok to me, but I'm happy to incorporate improvements in a future pr